histomicstk/cli/SuperpixelSegmentation/SuperpixelSegmentation.py

```diff
@@ -0,0 +1,225 @@
+import json
+import logging
+import math
+import os
+
+import large_image
+import numpy
+import pyvips
+import scipy
+import skimage
+from histomicstk.cli import utils
+from histomicstk.cli.utils import CLIArgumentParser
+
+logging.basicConfig()
```

Comment:
If you want to use logging rather than prints, then we should change all the print statements (or remove the logging module).

Reply:
I decided to go with prints.